.travis.yml

```diff
@@ -1,14 +1,10 @@
 language: node_js
+sudo: false
 
 node_js:
-  - '0.12'
-  - '0.10'
-  - '0.8'
-  - 'iojs'
-
-before_install:
-- '[ "${TRAVIS_NODE_VERSION}" != "0.8" ] || npm install -g npm@1.4.28'
-- npm install -g npm@2.12
+  - '6'
+  - '5'
+  - '4'
```

Comment:
Sorry if my earlier comment was unclear.

The list should be:

```
- '6'
- '5'
- '4'
- '0.12'
- '0.10'
```

Those are all the current supported versions of Node, which is what I'm trying to emulate.